jjb/anax/anax.yml

```diff
@@ -0,0 +1,21 @@
+- job:
+    name: jenkins-job
+    project-type: pipeline
+    defaults: global
+    node: 'ubuntu2204-builder-aws-1c-2g'
+    pipeline-scm:
+      script-path: "Jenkinsfile"
+      scm:
+        - git:
+            url: "https://github.com/open-horizon/anax.git"
+            clean: true
+    scm:
```

Comment:
Only pipeline-scm will be used by a pipeline job.

Reply:
how to specify the branch? pipeline-scm does not have "branch".

Reply:
pipeline-scm just accepts a regular scm, so you can add the branch to your git definition.